game-service/src/main/java/org/libertybikes/game/core/GameRound.java

```diff
@@ -172,6 +235,14 @@ public boolean addPlayer(Session s, String playerId, String playerName, Boolean
             isPhone = c.isPhone = hasGameBoard ? false : true;
             clients.put(s, c);
             log("Player " + playerId + " has joined.");
+
+            // Increment player counter metrics
+            getRegistry().counter(currentPlayersCounter).inc();
```

Comment:
inconsistent use of null checks here. If CDI is not enabled (unit test env?) or otherwise fail to lookup the metrics registry we will get a NPE here. I prefer some solution where we don't need to be doing null checks all over the place